src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
+.info {
+  margin: 20px 0;
+}
+
+.info__data {
+  margin: 10px 0;
```

Comment:
by convention, the distance between the elements should be 10 pixels in total, and you add vertical indents, that is, both for the top and for the bottom
fix it
![image](https://user-images.githubusercontent.com/70009205/232860481-7a46177d-0f4e-458f-942b-d978584577d7.png)